components/command/windowsOSCommand.go

```diff
@@ -12,6 +14,15 @@ var _ OSCommand = (*windowsOSCommand)(nil)
 
 type windowsOSCommand struct{}
 
+func (fs windowsOSCommand) NewCopyFile(runner *Runner, localPath, remotePath string, opts ...pulumi.ResourceOption) (*remote.CopyFile, error) {
```

Comment:
Is it expected that this function does not have the same behavior as on Unix? I think this one will not copy the file to a world-writable folder first and then copy it. Hence the `CopyRemoteFile` is not used for now

Reply:
Yes! For Windows we use `Administrator`, that has the permission to copy files on system folders. The problem only exists on Linux because we do not use `root`. I can update the description to make it explicit

Reply:
Maybe we could remove the `CopyRemoteFile` method then? Or should it be available to be used elsewhere?

Reply:
I can drop it for now I guess. At first I added it to the interface so I wrote both implementations. I can convert it to to a private function in the `unix` command. On it!